public/site.webmanifest

```diff
@@ -0,0 +1 @@
+{"name":"","short_name":"","icons":[{"src":"/android-chrome-192x192.png","sizes":"192x192","type":"image/png"},{"src":"/android-chrome-512x512.png","sizes":"512x512","type":"image/png"}],"theme_color":"#ffffff","background_color":"#ffffff","display":"standalone"}
```

Comment:
I don't need this file, let's remove it.

Reply:
Hmm, actually, keep it, this would be handy for phone shortcuts.

Reply:
Hello @mrozbarry, the favicons have been updated and the missing headers were added to public/timer.html

Reply:
Can you set name and short_name to `"mobtime"` please and thanks

Reply:
Yes sir, I can definitely do that